packages/backend/src/managers/playground.ts

```diff
@@ -294,7 +312,7 @@ export class PlayGroundManager {
         this.sendQueriesState();
       }
     })().catch((err: unknown) => console.warn(`Error while reading streamed response for model ${modelInfo.id}`, err));
-
+    this.telemetry.logUsage('playground.ask', { 'model.id': modelInfo.id });
```

Comment:
Maybe we should also record the execution time for a request but that could be a follow up PR